src/libs/PaymentUtils.js

```diff
@@ -64,6 +64,16 @@ function getPaymentMethods(bankAccountList, cardList, payPalMeUsername = '') {
     return combinedPaymentMethods;
 }
 
+/**
+ * Get the amount that is being transferred
+ * @param {Number} currentBalance
+ * @returns {Number}
+ */
+function getWalletTransferAmount(currentBalance) {
```

Comment:
This function does one thing (performs a calculation) and it doesn't look like a "getter" (i.e. PaymentUtils supplies the value). 

Can we please try to give it a name that describes **what it does** like `subtractWalletTransferBalanceFee(currentBalance)`?

Reply:
Ok. Sure.